packages/elements/src/photon-dropdown/photon-dropdown.tsx

```diff
@@ -214,6 +214,9 @@ export const PhotonDropdown = <T extends { id: string }>(props: {
           if (props.onOpen) {
             await props.onOpen();
           }
+          setTimeout(() => {
```

Comment:
is there a delay?

Reply:
ah well, it seems to fix on my computer's simulator but I'm not quite seeing it on my phone yet with the branch version, so testing out some timeouts